frontend/services/entities/team_policies.ts

```diff
@@ -159,7 +159,7 @@ export default {
   getCount: async ({
     query,
     teamId,
-    mergeInherited = false,
+    mergeInherited = true,
```

Comment:
Defaulting to true as the UI does not have any option (currently at least) to render `merge_inherited = false`